provisioning/ubuntu-provision.sh

```diff
@@ -156,7 +156,7 @@ function install_asdf() {
   rm -f "${archive}"
 
   ## append to the system wide path variable, need to be seconded for docker in packer sources.pkr.hcl
-  sed -e "/^PATH/s/\"$/:\${asdf_install_dir}\/shims:\${asdf_install_dir}\/bin\"/g" -i /etc/environment
+  sed -e '/^PATH/s/"$/:\/home\/jenkins\/.asdf\/shims:\/home\/jenkins\/.asdf\/bin\"/g' -i /etc/environment
```

Comment:
```suggestion
  sed --regexp-extended "s|^PATH=(.*)"$|PATH=\1:${asdf_install_dir}/shims:${asdf_install_dir}/bin"|" -i /etc/environment
```
Result of mob programming with @smerle33 @dduportal